### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in disability-rating-calculator end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7320]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. These tests should include thorough axe checks. While auditing the `/disability-benefits/disability-rating-calculator` app folder, I didn't see any axe checks in the e2e test file. I'd like the front-end engineering team to review this application, and add axe checks for the default state and ideally a state where 2-3 disabilities have been added. Definition of done in acceptance criteria below.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications/disability-benefits/disability-rating-calculator/tests`
* `$ yarn test:e2e src/applications/disability-benefits/disability-rating-calculator/tests`

## Acceptance Criteria

**Definition of done:**
- [x] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [x] axe checks are added for the default view
- [x] axe checks are added for a view where 2-3 disabilities are added and the rating calculator has run
- [x] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [x] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

**Comment by 1Copenut** [comment]
@mattpointzxer0 If the team has moved to use the Cypress forms testing, would you mind commenting here and at include me? The Cypress forms tester has axe checks built in and would be a good way to close this issue without additional work.

**Comment by 1Copenut** [comment]
Hi @brianalloyd. I'm taking a sweep through our e2e tests again to ensure good accessibility coverage. I wanted to check in briefly and see if the Disability Calculator had tests, or was being evaluated by axe in our static content page check. Thank you!

**Comment by brianalloyd** [comment]
@1Copenut thanks for circling back, TBH I will need to dig deeper into this and validate the Disability Calculator test activities.  I'll follow back up with you once we've done a quick assessment.  Thanks.

**Comment by ncksllvn** [comment]
Seems like we just need to add an Axe Check to this test and fix any a11y errors that may surface as a result, https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js

**Comment by ncksllvn** [comment]
FYSA that the calculator lives at `/disability/about-disability-ratings/`

**Comment by kelsonic** [comment]
PR is up: https://github.com/department-of-veterans-affairs/vets-website/pull/17022
